lib/tasks/clean_up_submissions.rake

```diff
@@ -0,0 +1,41 @@
+namespace :curate do
+  desc 'Purge all submissions and related records'
+  task purge_submissions: :environment do
+
+    ss = Submission.all
+    puts "Found #{ss.length} submissions."
+
+    ss.each do |s|
+      s.destroy
+    end
+
+    sus = Submission::Upload.all
+    puts "Found #{sus.length} submission uploads."
+
+    sus.each do |su|
+      su.destroy
+    end
+
+    models = ActiveRecord::Base.descendants
+
+    models.each do |m|
```

Comment:
You will remove `ApiKeys`. You can probably limit this to `Api.publishable_models`.